pulp/pulp.py

```diff
@@ -880,38 +1003,145 @@ def addInPlace(self, other, sign=1):
             self.constant += other * sign
         return self
 
-    def subInPlace(self, other):
+    def subInPlace(
+        self,
+        other: Union[
+            "LpAffineExpression",
+            Dict[LpVariable, Union[int, float]],
+            List[
+                Union["LpAffineExpression", LpVariable, "LpConstraintVar", int, float]
+            ],
+            Iterable[LpVariable],
+            LpVariable,
+            "LpConstraintVar",
+            int,
+            float,
+            None,
+        ],
+    ) -> "LpAffineExpression":
```

Comment:
this `other` is repeated in several places. (1) I'm not sure it's correct (2) if it is, we should create it once and re-use it